### PR TITLE
Make stuff portable

### PR DESCRIPTION
**Pull request by BrennanConroy**
@mikeharder @pakrym

### CODE REVIEW

File: src/Benchmarks.ClientJob/project.json [hunk 1]
```diff
@@ -13,9 +13,12 @@
     "Newtonsoft.Json": "8.0.3"
   },
   "frameworks": {
-    "netstandardapp1.5": {
```
Comment (rynowak) [inline src/Benchmarks.ClientJob/project.json]: Aren't `ClientJob`, `ServerJob` and `Repository` class libraries?

Comment (BrennanConroy) [inline src/Benchmarks.ClientJob/project.json]: Maybe, then they should have been targeting netstandard1.x instead of netstandardapp

Comment (mikeharder) [inline src/Benchmarks.ClientJob/project.json]: They are supposed to be class libraries, I authored the project.json wrong.  Can you change them now?

We should also add support for NetFx 4.5, I can add this in a later change.

Comment (BrennanConroy) [inline src/Benchmarks.ClientJob/project.json]: Netstandard1.0 :)

**Comment by mikeharder** [comment]
:shipit:

**Pull request merged**